html2text/utils.py

```diff
@@ -204,13 +204,24 @@ def skipwrap(para, wrap_links):
 
 
 def wrapwrite(text):
+    text = text.encode('utf-8')
+    
+    try:  # Python3
+        sys.stdout.buffer.write(text)
+    except AttributeError:
+        sys.stdout.write(text)
+
+
+def wrapwrite_surrogate_escape(text):
```

Comment:
Instead of making new function with almost identical body except passing "surrogateescape" to `text.encode`, couldn't be be used with a parameter passed to to the original `wrapwrite`?

For instance changing:

```python
def wrapwrite(text):
    text = text.encode('utf-8')
    ....
```

into
```python
def wrapwrite(text, errors='strict'):
    text = text.encode('utf-8', errors)
    ....
```